pkg/support/dump_test.go

```diff
@@ -38,3 +98,108 @@ func TestParseFileName(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, ts.String(), "2021-08-17 09:47:58 +0000 UTC")
 }
+
+func TestTimestampFilter(t *testing.T) {
+	ts := timestampFilter("2m")
+	currentTime := time.Now()
+	require.NotNil(t, ts)
+	expectedTime := currentTime.Add(-2 * time.Minute)
+	assert.WithinDuration(t, expectedTime, *ts, 10*time.Second)
+}
+
+func TestWriteFile(t *testing.T) {
+	basedir := "dir1/dir2"
+	resource := "networkpolicies"
+	data := []byte("antct -oyaml get networkpolicies")
+	expectedErr := "error when writing  to file: open :"
+	fs := afero.NewMemMapFs()
+	err := writeFile(fs, filepath.Join(basedir, resource), resource, data)
+	if err != nil {
+		assert.ErrorContains(t, err, expectedErr)
+	} else {
+		require.NoError(t, err)
```

Comment:
the way these tests (this one and subsequent ones) are written doesn't make sense:
```go
	if err != nil {
		assert.ErrorContains(t, err, expectedErr)
	} else {
		require.NoError(t, err)
        }
```
either you expect an error or you don't. There is only one test case here, so the branching doesn't make sense.